common/src/main/java/com/wynntils/core/functions/EnableableFunction.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.core.functions;
+
+public abstract class EnableableFunction<T> extends Function<T> {
```

Comment:
ActiveFunction?

Reply:
It's definitely less of a mouthful. 

Yeah, it might work. I'll try renaming it and we'll see if it looks ok.